public/app/features/browse-dashboards/state/hooks.ts

```diff
@@ -97,10 +104,15 @@ export function useActionSelectionState() {
   return useSelector((state) => selectedItemsForActionsSelector(state));
 }
 
-export function useLoadNextChildrenPage() {
+export function useLoadNextChildrenPage(
+  excludeKinds: Array<DashboardViewItemWithUIItems['kind'] | UIDashboardViewItem['uiKind']> = []
+) {
   const dispatch = useDispatch();
   const requestInFlightRef = useRef(false);
 
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const memoizedExcludeKinds = useMemo(() => excludeKinds, excludeKinds);
```

Comment:
this looks sus 👀 

why do we need to memoize this then just return the same thing anyway?! 🤔

Reply:
originally i declared the array inline, so it would be different every time, but then i hoisted it out. i left this in because i thought it would be safer, but maybe it doesn't matter?

Reply:
yeah i don't think it's doing anything now other than adding noise tbh 👍